src/TraceEvent/TraceLog.cs

```diff
@@ -2163,7 +2179,6 @@ private unsafe void CopyRawEvents(TraceEventDispatcher rawEvents, IStreamWriter
             // While scanning over the stream, copy all data to the file.
             rawEvents.AllEvents += delegate (TraceEvent data)
             {
-                Debug.Assert(_syncTimeQPC != 0);         // We should have set this in the Header event (or on session start if it is read time
```

Comment:
Do you know what changed `_syncTimeQPC` to be 0?  I see this in a few places in this change, but I don't see what actually changed it.

Reply:
Oops, that was the assert above. This is the line that changed the test data: https://github.com/microsoft/perfview/pull/2200/files#diff-c94b1f053e78f716a557a5876766fc48558133ec18fe0fa752e5cf40c23a6fcfR2850

Reply:
Gotcha, thanks.